### ISSUE TITLE
Document lib testing

### ISSUE THREAD

**Issue by ComLock** [enonic/doc-xp#47]
I know about these functions
* assertEquals
* assertJson
from https://developer.enonic.com/guides/writing-unit-tests

And I know there are more, but I can't even find where I saw them yesterday...

Nothing about testing here either: https://repo.enonic.com/public/com/enonic/xp/docs/7.0.2/docs-7.0.2-libdoc.zip!/index.html

**Comment by ComLock** [comment]
Here they are: https://github.com/enonic/xp/blob/master/modules/tools/testing/src/main/resources/lib/xp/testing.js

**Comment by jsi** [comment]
As far as I can tell, this issue has been solved with this guide: https://github.com/enonic/guide-testing-apps/blob/master/docs/index.adoc

The result is found here: https://developer.enonic.com/guides/writing-unit-tests

**Comment by alansemenov** [comment]
The task is to document [these API methods](https://github.com/enonic/xp/blob/master/modules/tools/testing/src/main/resources/lib/xp/testing.js) in [this guide](https://developer.enonic.com/guides/writing-unit-tests). Could be a separate section called "API" between "Add dependency" and "Testing a library"

**Comment by poi33** [comment]
They are shipped with core. So should they not be included as an test api. The same way portal, node and auditlog are included?
Not in a guide?

**Comment by thomasheartman** [comment]
I agree with what @poi33 is saying here 🙋🏾  When looking for them, the first place I went was to the XP reference. Not necessarily because I assumed the ships with XP, but because that's the place I thought was the most likely to have info about testing XP.

**Comment by poi33** [comment]
@rymsha could we document the testing api methods that where mentioned above?

**Comment by rymsha** [comment]
Yes you can.
